### PR TITLE
API warnings for services with bad combos of ExternalIPs and ExternalName

### PR DESCRIPTION
**Pull request by thockin**
/kind cleanup

```release-note
NONE
```

/sig network

### CODE REVIEW

File: pkg/api/service/warnings.go [hunk 1]
```diff
@@ -53,6 +53,13 @@ func GetWarningsForService(service, oldService *api.Service) []string {
 		warnings = append(warnings, getWarningsForCIDR(field.NewPath("spec").Child("loadBalancerSourceRanges").Index(i), cidr)...)
 	}
 
+	if service.Spec.Type == api.ServiceTypeExternalName && len(service.Spec.ExternalIPs) > 0 {
```
Comment (khenidak) [inline pkg/api/service/warnings.go:56]: please add tests.

Comment (thockin) [reply pkg/api/service/warnings.go:56]: added !

**Comment by thockin** [comment]
@kubernetes/release-managers  I'd like to get this in for 1.28.  Very low risk.

**Comment by xmudrii** [comment]
@thockin This is up to @kubernetes/release-team-leads to decide given that this is for an ongoing release cycle 🙂

**Comment by gracenng** [comment]
That's fine with me 

@furkatgofurov7 pls track this as well!

**Comment by dims** [comment]
needs milestone to land

**Comment by khenidak** [comment]
/lgtm /approve

thanks

**Comment by furkatgofurov7** [comment]
> That's fine with me
> 
> @furkatgofurov7 pls track this as well!

Can someone please milestone this with v1.28 to track it?

**Comment by xmudrii** [comment]
@furkatgofurov7 Sure!
/milestone v1.28

**Comment by thockin** [comment]
Thanks!

**Pull request merged**